Update bank.json  adding honduran banks

rzelaya11 opened the pull request.

bhousel commented on brands/amenity/bank.json:
```diff
@@ -3459,6 +3468,17 @@
       "name": "Federal Bank"
     }
   },
+  "amenity/bank|Ficohsa": {
```
For this one, I notice:
* It operates in countries other than Honduras. (see https://www.ficohsa.com/)
* `brand` and `name` tags should just be "Ficohsa"
* need to replace underscores with spaces in the `brand:wikipedia` tag
* remove the redundant matchName

bhousel commented on brands/amenity/bank.json:
```diff
@@ -5170,6 +5190,17 @@
       "name": "Privatbanka"
     }
   },
+  "amenity/bank|Promerica": {
```
For this one I notice:
* It operates in countries other than just Honduras (I found https://www.promerica.fi.cr/, not sure how these brands are related)
* identifier and `brand` should probably consistently be either "Promerical" or "Banco Promerica" (with matchName for whatever the less preferred alternative is)
* need to replace underscores with spaces in the `brand:wikipedia` tag

bhousel commented on brands/amenity/bank.json:
```diff
@@ -2254,6 +2254,15 @@
       "name": "Banorte"
     }
   },
+  "amenity/bank|Banpais": {
```
For this one, it would be great to have a wikidata link.  Also please remove the redundant `matchNames` property.

bhousel commented:
Thanks @rzelaya11 and welcome to the project 👋 
I ended up addressing all the comments in d97af970 because I want to clear out the PR backlog.  

Please keep up the work of adding the Honduran brands that you know about - we always need more volunteers with local knowledge 👍

rzelaya11 commented:
Thanks for the comments, i have been a Little busy sorry i havent make the changes, i Will continue adding more brands and i Will make those corrections soon.

Enviado desde Correo<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows 10

De: Bryan Housel<mailto:notifications@github.com>
Enviado: martes 02 de junio de 2020 21:06
Para: osmlab/name-suggestion-index<mailto:name-suggestion-index@noreply.github.com>
CC: rzelaya11<mailto:r_zelyam@hotmail.com>; Mention<mailto:mention@noreply.github.com>
Asunto: Re: [osmlab/name-suggestion-index] Update bank.json adding honduran banks (#3928)


Thanks @rzelaya11<https://github.com/rzelaya11> and welcome to the project 👋
I ended up addressing all the comments in d97af97<https://github.com/osmlab/name-suggestion-index/commit/d97af970f76b46b4146af802667b9bfc9778dca5> because I want to clear out the PR backlog.

Please keep up the work of adding the Honduran brands that you know about - we always need more volunteers with local knowledge 👍

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/osmlab/name-suggestion-index/pull/3928#issuecomment-637925011>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AFOTQFWJOZRO6DQI7EVCN4DRUW42PANCNFSM4NQCZG2A>.
